File: sandbox/node/src/AccountGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.AccountApi();


apiCaller.accountGet().then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Account#accountGet:");
  console.log(error.body);
});

File: sandbox/python/src/AccountGetDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.AccountApi(api_client).account_get()

        pprint(response)
    except ApiException as e:
        print("Exception when calling Account#account_get: %s\n" % e)
